components/renku_data_services/notebooks/api.spec.yaml

```diff
@@ -1015,19 +1015,15 @@ components:
           additionalProperties: true
         readonly:
           type: boolean
-          default: true
         source_path:
           type: string
         target_path:
           type: string
         storage_id:
           allOf:
             - "$ref": "#/components/schemas/Ulid"
-            - description: The storage ID is used to know which storage config from the DB should be overriden
+            - description: If the storage_id is provided then this config must replace an existing storage config in the session
       required:
-        - configuration
-        - source_path
-        - target_path
         - storage_id
```

Comment:
Note: it would probably make sense to update `storage_id` to be `data_connector_id` in the future.

Reply:
I did not want to change the api on purpose. But yes we should do this in the future.

Reply:
I opened an issue #494